src/views/SimpleAccount/DepositLists.tsx

```diff
@@ -38,24 +38,26 @@ export const SimpleV4DepositList: React.FC<{ usersAddress: string }> = (props) =
 }
 
 /**
- *
+ * Displays V3 deposits (hidden after loading if empty).
  * @param props
  * @returns
  */
 export const SimpleV3DepositList: React.FC<{ usersAddress: string }> = (props) => {
   const { usersAddress } = props
-  const { data } = useUsersV3PrizePoolBalances(usersAddress, false)
+  const { data, isFetched } = useUsersV3PrizePoolBalances(usersAddress)
   const { t } = useTranslation()
-  return (
-    <div>
-      <CardTitle
-        className='mb-2'
-        title={`V3 ${t('deposits')}`}
-        secondary={`$${data?.totalValueUsd.amountPretty || '0.00'}`}
-      />
-      <V3DepositsList usersAddress={usersAddress} />
-    </div>
-  )
+  if(!isFetched || data.balances.length > 0) {
```

Comment:
Rather than returning `undefined` we should explicitly check first and have the table be the fallback return value.
```
if (!isFetched || data.balances.length === 0) return null
```

Reply:
Fixed through d8b2b5b